app/views/listings/show.html.erb

```diff
@@ -8,7 +20,7 @@
          guests
          <% else %>
          guest
-         <% end %> . <%= @listing.no_of_rooms %> 
+         <% end %> . <%= @listing.no_of_rooms %>
         <% if @listing.no_of_rooms > 1%>
          rooms
```

Comment:
consider using the following helper method for pluralization
[pluralize](https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize)(1, 'person')